sdk/test-utils/perf/src/multicore.ts

```diff
@@ -0,0 +1,168 @@
+import { ChildProcess, fork } from "child_process";
+import workerThreads from "worker_threads";
+import {
+  ManagerToWorkerMessage,
+  WorkerToManagerMessage,
+  WorkerToManagerMessageWithId,
+} from "./messages";
+import {
+  defaultPerfOptions,
+  DefaultPerfOptions,
+  ParsedPerfOptions,
+  parsePerfOption,
+} from "./options";
+
+export type WorkerLike = workerThreads.Worker | ChildProcess;
+
+export interface WorkerData {
+  testClassName: string;
+  assignedParallels: number;
+  parallelIndexOffset: number;
```

Comment:
These two fields might use some docs or else one has to look deeper to find out what they're used for :P